source/constructs/config/job/script/glue-job.py

```diff
@@ -65,8 +65,39 @@ class ColumnDetector:
     """
     ColumnDetector is used to detect entities in a column.
     """
-    def __init__(self, broadcast_template):
+    def __init__(self, broadcast_template, job_exclude_keywords):
         self.broadcast_template = broadcast_template
+
+        raw_job_exclude_keywords = job_exclude_keywords.split(',') if job_exclude_keywords else []
+        self.job_exclude_keywords = list(filter(None, raw_job_exclude_keywords))
+    
+    def validate_column_name(self, column_name, identifier_include_keywords, identifier_exclude_keywords):
+        """
+        validate_column_name is used to check whether a column name is valid.
+        A column name is valid when it is not in the exclude keywords list.
+
+        Args:
+            column_name: The name of a column.
+        
+        Returns:
+            True if column_name is valid, False otherwise.
+        """
+        valid_column_header = False
+        
+        total_exclude_keywords = identifier_exclude_keywords + self.job_exclude_keywords
+        for exclude_keyword in total_exclude_keywords:
+            if re.search(exclude_keyword, column_name):
+                valid_column_header = False
+                return valid_column_header
+
+        if not identifier_include_keywords or len(identifier_include_keywords) == 0:
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

To check if a container or sequence (string, list, tuple) is empty, use `if not val`. Do not compare its length using `if len(val) == 0`

[Learn more](https://www.python.org/dev/peps/pep-0008/#programming-recommendations#:~:text=if%20not%20seq)